Src/Plugins/ValidatorBase.cs

```diff
@@ -113,6 +119,11 @@ protected ValidatorBase()
             return validationState;
         }
 
+        public static void DisableValidationCaching(bool disableValidationCaching)
```

Comment:
>DisableValidationCaching [](start = 27, length = 24)

Just use a static property for this!

Reply:
changing to a property or field is a problem. We would need the object to change and we don't have it.
I will leave as a method for now and we can try later to change again

---
In reply to: [592834390](https://github.com/microsoft/sarif-pattern-matcher/pull/288#discussion_r592834390) [](ancestors = 592834390)